pombola/nigeria/views.py

```diff
@@ -164,29 +72,24 @@ def get_context_data(self, **kwargs):
             context['governor'] = self.find_governor(context['state'])
 
             # work out the polygons to match to, may need to go up tree to parents.
-            area_for_polygons = self.find_containing_area(area)
-            if area_for_polygons:
-                area_polygons = area_for_polygons.polygons.collect()
-
-                context['federal_constituencies'] = self.get_district_data(
-                    self.find_matching_places("FED", area_polygons),
-                    "representative"
-                )
-
-                context['senatorial_districts']  = self.get_district_data(
-                    self.find_matching_places("SEN", area_polygons),
-                    "senator"
-                )
+            # area_for_polygons = self.find_containing_area(area)
+            # if area_for_polygons:
+            #     area_polygons = area_for_polygons.polygons.collect()
+
+            #     context['federal_constituencies'] = self.get_district_data(
+            #         self.find_matching_places("FED", area_polygons),
+            #         "representative"
+            #     )
+
+            #     context['senatorial_districts']  = self.get_district_data(
+            #         self.find_matching_places("SEN", area_polygons),
+            #         "senator"
+            #     )
         return context
 
     def parse_params(self):
         super(NGSearchView, self).parse_params()
-        tidied_as_if_pun = tidy_up_pun(self.query)
-        # Check if this is a known form of PUN:
-        if pun_re.search(tidied_as_if_pun):
-            self.pun = tidied_as_if_pun
-        else:
-            self.pun = None
+        self.pun = self.query
```

Comment:
This logic needs to be changed slightly so we don't always assume we're dealing with a PU number.